apps/scan/backend/src/export.ts

```diff
@@ -1,13 +1,63 @@
 import { LogEventId, Logger } from '@votingworks/logging';
-import { ExportCastVoteRecordsToUsbDriveError } from '@votingworks/types';
 import {
+  ExportCastVoteRecordsToUsbDriveError,
+  SystemSettings,
+} from '@votingworks/types';
+import {
+  assert,
+  assertDefined,
   extractErrorMessage,
+  iter,
   Result,
   throwIllegalValue,
 } from '@votingworks/basics';
-import { exportCastVoteRecordsToUsbDrive as exportCastVoteRecordsToUsbDriveBackend } from '@votingworks/backend';
+import {
+  exportCastVoteRecordsToUsbDrive as exportCastVoteRecordsToUsbDriveBackend,
+  Sheet,
+} from '@votingworks/backend';
 import { UsbDrive } from '@votingworks/usb-drive';
+import { encryptAes256 } from '@votingworks/auth';
 import { Workspace } from './util/workspace';
+import { Store } from './store';
+
+export async function encryptBallotAuditId(
+  store: Store,
+  systemSettings: SystemSettings,
+  sheet: Sheet
+): Promise<Sheet> {
+  if (
+    !(
+      systemSettings.precinctScanEnableBallotAuditIds &&
+      sheet.type === 'accepted' &&
```

Comment:
My understanding is murky here, but wondering why we wouldn't encrypt the audit ID for non-`accepted` CVRs - do those get exported to the USB drive as well? And if they do, is there any concern with those being plain text? (Assuming that even if they're not tied to a tallied CVR, they at least have some record of attempted votes)

Reply:
I don't think rejected ballots get audited, so I didn't think there was any reason to include the audit ID. Also, I don't think all of them would have the audit ID in the metadata anyway depending on how far along they got in interpretation.